### DISCUSSION TITLE
Issue with failing to deploy, error called run mkdocs gh - deploy --force

### DISCUSSION THREAD

**Discussion by nearysam** [giswqs/geohey#12] [Q&A] [answered]
I corrected the previous codespell issue by adding to execute ignore and docs build/yml codespell order however now I am getting another error called run "mkdocs gh - deploy -- force". Is there something I did not do correctly with fixing the previous error or is this something unrelated?

**Comment by giswqs** [comment, marked as answer]
Make sure you check the file path carefully. It should be `execute_ignore: ["examples/lab5.ipynb, examples/lab4.ipynb"]`. The error message shows you why it failed. 
https://github.com/nearysam/springburn/blob/main/mkdocs.yml#L53
